plugins/extensions/plugins/DeflectPlugin.cpp

```diff
@@ -71,8 +72,9 @@ namespace brayns
         '*', "Enable/Disable Deflect streaming",
                 [&] { _params.setEnabled( !_params.getEnabled( )); });
 
-    // ensure streaming is off initially, otherwise our logic in run() won't work
-    _params.setEnabled( false );
+    _params.setEnabled( true ); // Streaming will only be activated if
```

Comment:
I hope you tested it from HTTP then, because I did put that comment on purpose from what I remember.